src/routes/allOffer/allRideOffer.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import { rideOffers } from '../../controllers/rideOffers/rideOffers';
+
+
+export const allRideOfferRouter = express.Router();
+
+allRideOfferRouter.get('/all-ride-offer', rideOffers);
```

Comment:
Please check the link I sent for proper API route naming, this is already a get request a generic name would be `/offers`